samples/excel/89-preview-apis/protect-data-in-worksheet-and-workbook-structure.yaml

```diff
@@ -0,0 +1,258 @@
+                await worksheetHelper(context, false, true, await passwordHandler());
+            });
+        }
+
+        async function passwordProtectWorkbookStructure() {
+            await Excel.run(async (context) => {
+                await workbookHelper(context, true, true, await passwordHandler());
+            });
+        }
+
+        async function passwordUnprotectWorkbookStructure() {
+            await Excel.run(async (context) => {
+                await workbookHelper(context, false, true, await passwordHandler());
+            });
+        }
+
+        async function worksheetHelper(context: Excel.RequestContext, expectProtection: boolean, expectPassword: boolean, password?: string) {
+
+            await passwordHelper(expectPassword, password);
+            await Excel.run(async (context) => {
+                let activeSheet = context.workbook.worksheets.getActiveWorksheet();
+                activeSheet.load("protection/protected");
+
+                await context.sync();
+
+                if (expectProtection && !activeSheet.protection.protected) {
+                    activeSheet.protection.protect(null, password);
+                } else if (!expectProtection) {
+                    activeSheet.protection.unprotect(password);
+                }
+
+                activeSheet.load("name");
+                activeSheet.load("protection/protected");
+
+                await context.sync();
+
+                console.log("Expected: Worksheet '" + activeSheet.name + "' should " + await messageHelper(expectProtection) + ". Protected? " + activeSheet.protection.protected);
+            });
+        }
+
+        async function workbookHelper(context: Excel.RequestContext, expectProtection: boolean, expectPassword: boolean, password?: string) {
+
+            await passwordHelper(expectPassword, password);
+            await Excel.run(async (context) => {
+                let workbook = context.workbook;
+                workbook.load("protection/protected");
+
+                await context.sync();
+
+                if (expectProtection && !workbook.protection.protected) {
+                    workbook.protection.protect(password);
+                } else if (!expectProtection) {
+                    workbook.protection.unprotect(password);
+                }
+
+                workbook.load("protection/protected");
+
+                await context.sync();
+
+                console.log("Expected: This workbook should " + await messageHelper(expectProtection) + ". Protected? " + workbook.protection.protected);
+            });
+        }
+
+        async function messageHelper(expectProtection: boolean): Promise<string> {
+            
+            let message = "not be protected";
+            if (expectProtection) {
+                message = "be protected";
+            }
+
+            return message;
+        }
+
+        async function passwordHelper(expectPassword: boolean, password?: string) {
```

Comment:
You have the `async `keyword on this passwordHelper definition, but it doesn't look like it contains anything awaitable. I don't think the `async `is needed.